_config.yml

```diff
@@ -72,3 +84,63 @@ defaults:
       path: engine/reference/commandline
     values:
       skip_read_time: true
+
+# Fetch upstream resources (reference documentation) used by _plugins/fetch_remote.rb
+# - repo is the GitHub repository to fetch from
+# - ref the Git reference
+# - paths is a list to the resources within the remote repository
+#   - dest is the destination path within the working tree
+#   - src is a list of glob source paths within the remote repository
+fetch-remote:
+  - repo: docker/cli
+    ref: refs/heads/20.10
+    paths:
+      - dest: engine/extend
+        src:
+          - "docs/extend/**"
+          - "!docs/extend/cli_plugins.md" # the cli plugins api is not a stable API, and not included in the TOC for that reason.
```

Comment:
The `cli_plugins.md` file doesnt exist in the upstream repo anymore. Maybe we can remove this line? https://github.com/docker/cli/tree/20.10/docs/extend

Reply:
Good point, I just blindly translated the old shell script.